art/estimators/certification/derandomized_smoothing/derandomized_smoothing.py

```diff
@@ -0,0 +1,445 @@
+# MIT License
+#
+# Copyright (C) The Adversarial Robustness Toolbox (ART) Authors 2022
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+# persons to whom the Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+# Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+# WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""
+This module implements (De)Randomized Smoothing for Certifiable Defense against Patch Attacks
+
+| Paper link: https://arxiv.org/abs/2002.10733
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from abc import ABC, abstractmethod
+
```

Comment:
```suggestion
from abc import ABC, abstractmethod
```

Reply:
Hi Beat, it looks like there aren't any suggested changes here.

Reply:
Sorry, I wanted to suggest to remove line 27 between the imports of `abc` and `typing` to have them in one block.